build.sh

```diff
@@ -18,7 +18,7 @@ EOF
 	done
 }
 
-check_for_application lex yacc autoheader aclocal automake autoconf
```

Comment:
This only checks if there is an executable named `yacc`, which *bison* should also provide. Please don't make this *bison* specific as collectd should also build with other implementations of yacc.